hazelcast/src/main/java/com/hazelcast/util/StringUtil.java

```diff
@@ -247,4 +253,38 @@ public static int lastIndexOf(String input, char ch) {
             return null;
         }
     }
+
+    /**
+     * Convert getter into a property name
+     * Example: 'getFoo' is converted into 'foo'
+     *
+     * It's written defensively, when output is not a getter then it
+     * return the original name.
+     *
+     * It converters name starting with the get- prefix only. When a getter
+     * starts with is- prefix (=boolean) then it does not convert it.
+     *
+     * @param getterName
+     * @return property matching the given getter
+     */
+    public static String getterIntoProperty(String getterName) {
```

Comment:
my original choice was something like `stringGetterPrefix()`. but there is more going on - e.g. conversion to lower case. 

bear in mind this is located in a class `StringUtil`. do you have a better name?